pkg/sql/lease.go

```diff
@@ -78,9 +80,12 @@ func (s *LeaseState) Expiration() time.Time {
 	return s.expiration.Time
 }
 
-// hasSomeLifeLeft returns true if the lease has at least a minimum of lifetime
-// left until expiration, and thus can be used.
+// hasSomeLifeLeft returns true if the lease has at least a minimum of
+// lifetime left until expiration, and thus can be used.
 func (s *LeaseState) hasSomeLifeLeft(clock *hlc.Clock) bool {
+	if s.testingKnobs.CanUseExpiredLeases {
```

Comment:
FYI, I can also set MinLeaseDuration to a large negative number to achieve the same.